src/java/com/puppetlabs/http/client/impl/JavaClient.java

```diff
@@ -57,8 +58,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
Probably good to expand all of the individual imports used here rather than '*'.